Add favicon

Erstus opened the pull request.

Erstus committed:
Merge pull request #1 from 1Hyena/master

Update my fork

Erstus committed:
Add logo files.

title(.svg) & favicon(.gif)

Erstus committed:
Add favicon

Erstus committed:
fix typo of favicon

missing "

Erstus committed:
Replace title with svg

Erstus committed:
align version number with title.

1Hyena commented:
@Erstus 
excellent work, but one little thing ... .info is missing from the title :D 
so instead of `Cryptograffiti` please make it `Cryptograffiti.info`
this would make it state more explicitly what service the logo is referring to

Erstus committed:
Better title positioning.

No margins needed.

Erstus committed:
Title with less whitespace.

Erstus committed:
Align version number better.

No margin needed.

Erstus committed:
Make title with fluid.

This keeps the version number on the same line with title even if the screen is very narrow.

Erstus committed:
Title default height 32px.

Erstus committed:
Add hint message to title image on hovering.

Erstus commented:
Ok. Sure thing. I added the ".info" part and did other improvements as well.
